templates/jenkins.conf.j2

```diff
@@ -1,16 +0,0 @@
-server {
```

Comment:
Just a reminder that we'll want to increment the major version of this role now that we've broken backwards compatibility.